### ISSUE TITLE
[Design System] Create a tabs component

### ISSUE THREAD

**Issue by macfarlandian** [Recidiviz/web-libraries#25]
**What needs to be done? Why does it need to be done?**
Create a basic component that provides a panel of tabs. Should be WAI-ARIA compliant and keyboard navigable. 

Example mock:

![DhYxmjlA](https://user-images.githubusercontent.com/5385319/119575445-ec631980-bd6b-11eb-9e05-9f5f8b59d0d6.png)
